jsx/StaticDataTable.d.ts

```diff
@@ -0,0 +1,60 @@
+import {ReactNode} from 'react';
+/**
+ * This file contains a sufficient typescript definition of the StaticDataTable
+ * to allow it to be imported into TypeScript with "strict" mode and not trigger
+ * errors.
+ */
+
+type TableRow = (string|null)[];
+
+type StaticDataTableProps = {
+    Headers: string[],
+    Data: TableRow[],
+    RowNumLabel: string?,
+    getFormattedCell: (
+        columnname: string,
+        celldata: string,
```

Comment:
Can it be of other types?

Reply:
I don't know because code doesn't currently have explicit types, the rows are just "array" according to PropTypes, but I'd prefer to start the types as strict as possible and then loosen it if necessary.